tests/baselines/reference/cloduleTest2.errors.txt

```diff
@@ -47,7 +47,7 @@ tests/cases/compiler/cloduleTest2.ts(36,10): error TS2554: Expected 1 arguments,
         r.foo();
         r.bar(); // error

```

Comment:
@DanielRosenwasser I'm thinking about the correctness of using this message for call expressions. What do you think? Is this acceptable message?